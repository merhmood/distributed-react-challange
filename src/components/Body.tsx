import React from "react";
import "../assets/css/body.css";

type BodyProps = {
  title: string;
};

const Body = ({ title }: BodyProps) => {
  return (
    <div className="body">
      <h1>{title}</h1>
    </div>
  );
};

export default Body;
