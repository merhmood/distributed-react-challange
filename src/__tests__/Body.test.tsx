import React from "react";
import renderer from "react-test-renderer";
import Body from "../components/Body";

test("should render appropriate props in h1", () => {
  const tree = renderer.create(<Body title="Home" />).toJSON();
  expect(tree).toMatchSnapshot();
});
