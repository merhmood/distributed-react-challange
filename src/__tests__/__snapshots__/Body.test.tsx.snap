// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render appropriate props in h1 1`] = `
<div
  className="body"
>
  <h1>
    Home
  </h1>
</div>
`;
